test/test.js

```diff
@@ -584,20 +586,61 @@ exercise.forEach(function(exerciseMode) {
 
     // https://fetch.spec.whatwg.org/#response-class
     suite('Response', function() {
-      test('default status is 200 OK', function() {
+      featureDependent(test, emptyDefaultStatusText, "default status is 200 ''", function() {
```

Comment:
The empty `''` quotes look weird in the test description text. I think you can safely remove them